tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -633,7 +632,8 @@ public static VerifiableConsumer createFromArgs(ArgumentParser parser, String[]
             brokerHostandPort = res.getString("brokerList");
         } else {
             parser.printHelp();
-            Exit.exit(0);
+            //Can't use `Exit.exit` here because `Exit` doesn't exists on 0.8.2.2.
```

Comment:
nit: `doesn't exist until 0.11.0.0` to be more accurate? Ditto below.

Reply:
Fixed.